CHANGELOG.next.asciidoc

```diff
@@ -173,6 +173,7 @@ automatic splitting at root level, if root level element is an array. {pull}3415
 - Fix metadata enricher with correct container ids for pods with multiple containers in container metricset. Align `kubernetes.container.id` and `container.id` fields for state_container metricset. {pull}34516[34516]
 - Make generic SQL GA {pull}34637[34637]
 - Collect missing remote_cluster in elasticsearch ccr metricset {pull}34957[34957]
+- Fix no error logs displayed in CloudWatch EC2, RDS and SQS metadata {pull}35035[35035]
```

Comment:
One more thing: the changelog allows mentioning an issue alongside the PR.

In this case, it would be something like this:

```suggestion
- Fix no error logs displayed in CloudWatch EC2, RDS and SQS metadata {issue}34985[34985] {pull}35035[35035]
```